src/com/yetanalytics/lrs_admin_ui/views/accounts.cljs

```diff
@@ -35,7 +36,15 @@
               [:a {:href "#!"
                    :on-click #(swap! delete-confirm not)
                    :class "icon-delete"}
-               "Delete"]])]]]]])))
+               "Delete"]])
+           (when (= username current-username)
+             [:li
+              [:a {:href "#!"
+                   :on-click (fn [e]
+                               (fns/ps-event e)
+                               (dispatch [:session/set-page :update-password]))
+                   :class "icon-edit"}
+               "Update Password"]])]]]]])))
```

Comment:
Will want to call this something more general when we add more stuff to the account page, but for now it's fine.

Reply:
Since we want to keep local account usage at a minimum I don't know if that day will ever come, but sure.